tests/unit/test_dataset.py

```diff
@@ -80,10 +72,7 @@ def ner_utils_test_data(scope="module"):
 
 def test_maybe_download():
     # ToDo: Change this url when repo goes public.
-    file_url = (
-        "https://raw.githubusercontent.com/Microsoft/Recommenders/"
-        "master/LICENSE"
-    )
+    file_url = "https://raw.githubusercontent.com/Microsoft/Recommenders/" "master/LICENSE"
```

Comment:
are you sure this works? shouldn't be 
`file_url = "https://raw.githubusercontent.com/Microsoft/Recommenders/master/LICENSE"`

Reply:
Broken by autoformatting. Fixed and replaced with NLP license link